Command/DumpCommand.php

```diff
@@ -29,7 +29,20 @@ protected function configure()
                     'Override the target directory to dump JS translation files in.'
                 ),
             ))
-            ->setDescription('Dumps all JS translation files to the filesystem');
+            ->setDescription('Dumps all JS translation files to the filesystem')
+            ->addOption(
+                'format',
+                null,
+                InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
+                'If set, only the passed formats will be generated',
+                array()
+            )
+            ->addOption(
+                'merge',
```

Comment:
this should maybe be named ``merge-domains``, as you may as well merge locales in the future